index.md

```diff
@@ -11,18 +11,37 @@ Our open source community strives to:
-  * Advocating for, or encouraging, any of the above behavior.
-  * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+* **Be careful in the words that you choose**: we are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren't acceptable.
 * **When we disagree, try to understand why**: Disagreements, both social and technical, happen all the time. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of our community comes from its diversity, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
 
-This code is not exhaustive or complete. It serves to distill our common understanding of a collaborative, shared environment, and goals. We expect it to be followed in spirit as much as in the letter.
+## Definitions
+
+Harassment includes, but is not limited to:
+
+- Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, race, age, regional discrimination, political or religious affiliation
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment
+- Deliberate misgendering or use of ‘dead’ or rejected names
+- Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop
+- Threats of violence, both physical and psychological
+- Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm
+- Deliberate intimidation
+- Stalking or following
+- Harassing photography or recording, including logging online activity for harassment purposes
+- Sustained disruption of discussion
+- Unwelcome sexual attention, including gratuitous or off-topic sexual images or behaviour
+- Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect others from intentional abuse
+- Publication of non-harassing private communication
+
+Our open source community prioritizes marginalized people’s safety over privileged people’s comfort. We will not act on complaints regarding:
+
+- ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
```

Comment:
> This "reverse-ism" thing is based primarily on assumption that in all the communities across the globe it is, lets say, western white man who has the privilege and in general, a white man tends to live happy, full life while everyone else's is just struggle for survival.

Reverse racism and reverse sexism don't exist because racism and sexism are institutional forms of oppression. Context is everything, and the open source community, which is what this Code of Conduct is for (please don't derail), is [overwhelmingly white and male](http://readwrite.com/2013/12/11/open-source-diversity). Case in point: check out the [list of contributors](https://github.com/todogroup/opencodeofconduct/graphs/contributors) to this project.

> However, I believe large parts of Asia, Middle east, Africa and Latino-America will probably disagree with you - especially in some culturally established matriarchies.

This is a cool story. Thank you. Again, context is everything, so discrimination against white men in these communities, if it happened, wouldn't be reverse racism or reverse sexism. It would be racism and sexism, because things like political boundaries and cultural power structures are institutions, and racism and sexism are institutional forms of oppression. However, appealing to examples of matriarchies is derailing this conversation. The open source community is overwhelmingly white and male, so cases of discrimination against women or persons of color (and all the other usual [axes of oppression](http://judge-me-not.weebly.com/uploads/1/0/4/4/10440246/8561435_orig.jpg)) are particularly prevalent.

> In addition, this is a review. Please, refrain yourself from making it personal in this or any other way, 

[Derailing (using emotion)!](http://www.derailingfordummies.com/derail-using-emotion/)

> and instead of appealing the reviewer, argue against the argument. Everybody you meet here does not come from utopian patriarchal society ruled by a stereotypical white male having every white male essentially spending life drinking mojitos and getting a tan.

You're right, everyone doesn't come from the same place, but this isn't a code of conduct for where people come from, it's a code of conduct for *this space*, which is open source software, which is overwhelmingly white and male (and other things too). Context is everything!